bndtools.core/bndtools.gtk.linux.x86_64.bndrun

```diff
@@ -1,15 +1,24 @@
 -include: bndtools.shared.bndrun
 
 -runbundles.linux: \
+	org.eclipse.swt.gtk.linux.x86_64,\
+	org.eclipse.e4.ui.workbench.renderers.swt,\
+	org.eclipse.equinox.security.linux.x86_64,\
 	org.eclipse.core.filesystem.linux.x86_64,\
-	org.eclipse.core.net.linux.x86_64,\
-    org.eclipse.equinox.security.linux.x86_64,\
-	org.eclipse.swt.gtk.linux.x86_64
+	org.eclipse.core.net.linux.x86_64
 
--runvm: ${crossplatform.vmargs}
+-runvm: \
+	-Dosgi.dataAreaRequiresExplicitInit=true \
+	-Dosgi.requiredJavaVersion=1.8 \
+	-XX:+UseG1GC \
+	-XX:+UseStringDeduplication \
+	${crossplatform.vmargs}
 
 -runproperties: \
 	${crossplatform.runprops},\
 	osgi.ws=gtk,\
 	osgi.arch=x86_64,\
 	osgi.os=linux
+
+-runblacklist.linux: \
+	osgi.identity;filter:='(osgi.identity=*macosx*)'
```

Comment:
Shouldn't `*win32*` also be in here?

Reply:
added for completeness though it was never getting resolved.